src/components/modules/auth/wallet/hooks/useWalletBalance.hook.ts

```
@@ -0,0 +1,264 @@
+
+      // Validate account structure
+      if (!account || !account.balances || !Array.isArray(account.balances)) {
+        console.error("Invalid account structure:", account);
+        throw new Error("Invalid account structure - useWalletBalance hook");
+      }
+
+      // Find ANY USDC asset balance (regardless of issuer)
+      const usdcBalance = account.balances.find(
+        (balance) =>
+          balance.asset_type === "credit_alphanum4" &&
+          balance.asset_code === "USDC"
```

ESLint

[error] Insert `,`

(prettier/prettier)